packages/@aws-cdk/aws-chatbot/README.md

```diff
@@ -34,10 +34,13 @@ const slackChannel = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-slackChannel.addLambdaInvokeCommandPermissions();
-slackChannel.addNotificationPermissions();
-slackChannel.addSupportCommandPermissions();
-slackChannel.addReadOnlyCommandPermissions();
+const invokeCommandStatement = new iam.PolicyStatement({
+  effect: iam.Effect.ALLOW,
+  resources: ['RESOURCE_ARNS_ALLOWED'],
+  actions: ['INVOKING_ACTION_ALLOWED'],
+});
+
+slackChannel.addToRolePolicy(invokeCommandStatement);
```

Comment:
I think we should remove this as well. If you want to include examples about adding template policies, it should be done in a separate snippet with corresponding documentation in a different PR.

Reply:
Ok, will remove this as well :)

Reply:
@MrArnoldPalmer Made the requested changes. Please review